gumby/modules/tribler_module.py

```diff
@@ -48,6 +48,13 @@ def set_transfer_size(self, size):
         self.transfer_size = long(size)
 
     @experiment_callback
+    def set_libtorrentmgr_alert_mask(self, mask=0xffffffff):
```

Comment:
I needed to see what libtorrent was doing. Tribler is very lax in processing libtorrent alerts. So this callback sets up the alert mask of libtorrent such that everything is dumped to the experiment log.